acceptance/auth/bundle_and_profile/test.toml

```diff
@@ -0,0 +1,5 @@
+Badness = "When -p flag is used inside the bundle folder for any CLI commands, CLI use bundle host anyway instead of profile one"
+
+[GOOS]
+# Disable this test on Windows because it's more complicated to set up .databrickscfg there
```

Comment:
Curious, what is the difficulty?

Reply:
There were some config path / home path related issues specific to Windows which were not obvious (CLI couldn't find .databrickscfg even though it was there). With a new approach it's not an issue anymore